inference-engine/tests/functional/plugin/cpu/single_layer_tests/normalize.cpp

```diff
@@ -90,10 +88,74 @@ const auto fusingMultiplyNoSharedChannel = fusingSpecificParams{std::make_shared
                 return std::make_shared<op::v1::Multiply>(inpNode, secondMultInput);
             }, "Multiply(NoSharedChannel)"}}), {"Multiply"}};
 
+const auto fusingAddSharedChannel = fusingSpecificParams{std::make_shared<postNodesMgr>(std::vector<postNodeBuilder>{
+            {[](std::shared_ptr<ngraph::Node> inpNode, const ngraph::element::Type& ngPrc, ngraph::ParameterVector& params){
+                SizeVector secondMultInShape(1, 1);
+                auto secondMultInput = builder::makeConstant(ngPrc, Shape(secondMultInShape), std::vector<float>{}, true);
+                return std::make_shared<op::v1::Add>(inpNode, secondMultInput);
+            }, "Add(SharedChannel)"}}), {"Add"}};
+
+const auto fusingAddNoSharedChannel = fusingSpecificParams{std::make_shared<postNodesMgr>(std::vector<postNodeBuilder>{
```

Comment:
Maybe `SharedChannel` -> `PerTensor` and `SharedChannel` -> `PerChannel` to align with others from fusing_test_utils.hpp? Also does it make sense to move them into common place?

Reply:
names changed, moved into fusing_test_utils.hpp